lambda/query-user-services/query-user-services.ts

```diff
@@ -1 +1,91 @@
-export const handler = async (): Promise<void> => {};
+import { SQSEvent, SQSRecord } from "aws-lambda";
+import { Service, TxmaEvent, UserData, UserRecordEvent } from "./models";
+import {
+  SendMessageCommand,
+  SendMessageRequest,
+  SQSClient,
+} from "@aws-sdk/client-sqs";
+import { DynamoDBDocumentClient, GetCommand } from "@aws-sdk/lib-dynamodb";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+
+const { TABLE_NAME, AWS_REGION } = process.env;
+const QUEUE_URL = "";
```

Comment:
Ooh I think we need to take the QUEUE_URL in from the environment too?
As this is, we'll pass an empty string to the [`sendSqsMessage` function ](https://github.com/alphagov/di-account-management-backend/pull/14/files#diff-387cce3998b99c626a6e32a1694ae07cd730e4e47ef928d58477c173e31ee1d3R85-R88)and that should make the amazon call a bit sad.

I think if this just goes:
```suggestion
const { TABLE_NAME, AWS_REGION, QUEUE_URL } = process.env;
```

Then you're all good as we'll give it the QUEUE_URL via cloudformation / amazon config.

Reply:
Scratch that, i see you just pushed a new commit will review that! 👀